### PR TITLE
chore(wallet): use next block conf target to estimate feerate

### PR DESCRIPTION
**Pull request by bradleystachurski**
Closes https://github.com/fedimint/fedimint/issues/5498

### CODE REVIEW

File: modules/fedimint-wallet-common/src/lib.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub mod txoproof;
 pub const KIND: ModuleKind = ModuleKind::from_static_str("wallet");
 pub const MODULE_CONSENSUS_VERSION: ModuleConsensusVersion = ModuleConsensusVersion::new(2, 1);
 
-pub const CONFIRMATION_TARGET: u16 = 10;
+pub const CONFIRMATION_TARGET: u16 = 1;
```
Comment (tvolk131) [inline modules/fedimint-wallet-common/src/lib.rs:38]: Can we also update this?

https://github.com/fedimint/fedimint/blob/6dd24f54dbad66edf2c99529523f5e3d2110e175/docs/wallet_module.md?plain=1#L25

Comment (tvolk131) [reply modules/fedimint-wallet-common/src/lib.rs:38]: Oh also, can you add a comment above this constant mentioning why the target is 1? I think just a brief mention of lack of RBF support and federation UTXOs being a shared resource would be useful.

**Review by dpc — DISMISSED** [review]

File: modules/fedimint-wallet-common/src/lib.rs (hunk 1 above)
Comment (bradleystachurski) [reply modules/fedimint-wallet-common/src/lib.rs:46]: Good call 👍

**Comment by bradleystachurski** [comment]
Looks like a blip with the runner while running back compat

> The self-hosted runner: runner-01-bb lost communication with the server

https://github.com/fedimint/fedimint/actions/runs/9761965930

**Pull request merged**